.devcontainer/devcontainer.json

```diff
@@ -21,9 +21,10 @@
 	// "remoteUser": "vscode",
 
 	// Add the IDs of extensions you want installed when the container is created in the array below.
-	// "extensions": [
+	"extensions": [
 	// 	"typescript.ts",
```

Comment:
Can we remove these example comments and keep just what we are actually using

Reply:
Done!